mod-password-validator/src/test/resources/domain/passwordvalidator/features/validate.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Test password validate
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapiToken)', 'Accept': 'application/json'  }
+
+  @Undefined
+  Scenario: POST validate should return 200 on success
```

Comment:
I think we should have several scenarios on 200:
- password is passing all enabled rules
- password is not passing a rule (for each default rule - separate scenario)

Reply:
Added new scenarios according to validation rules taken from our database

Reply:
Are you sure that if the password failed validation then we got 422 and not 200? As I see in API docs https://s3.amazonaws.com/foliodocs/api/mod-password-validator/s/password-validator.html it will be 200 for valid and not valid passwords.